front-end/src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -126,17 +128,18 @@ const VoteRefrendum = ({ className, referendumId, address, accounts, onAccountCh
 					? <div className={'LoaderWrapper'}>
 						<Loader text={loadingStatus.message}/>
 					</div>
-					: <>
+					: <Card>
 						<AccountSelectionForm
 							title='Vote with account'
 							accounts={accounts}
 							address={address}
+							balance={api && true}
```

Comment:
We can just call it `withBalance`. Not passing any value makes it equal to `true`:
```suggestion
							withBalance
```